index.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./style.css">
+    <title>Tribute</title>
+    <style>
+      #image{
+        max-width: 100%;
+        height: auto;
+      }
+      img{
+        display: block;
+      }
+    </style>
```

Comment:
I will suggest that you use an external style sheet because it makes it easier to maintain your webpage and track your code easily